packages/haiku-serialization/src/bll/Project.js

```diff
@@ -192,52 +186,39 @@ class Project extends BaseModel {
   handleMethodCall (method, params, message, cb) {
     return Lock.request(Lock.LOCKS.ProjectMethodHandler, false, (release) => {
       // Try matching a method on a given active component
-      const ac = (typeof params[0] === 'string')
-        ? this.findActiveComponentBySourceIfPresent(params[0])
-        : null
-
-      if (ac && typeof ac[method] === 'function') {
-        if (!SILENT_METHODS[method]) {
-          logger.info(
-            `[project (${this.getAlias()})] component handling method ${method}`
-          )
-        }
-
-        return ac[method].apply(ac, params.slice(1).concat((err, out) => {
-          release()
-
-          if (err) {
+      if (typeof params[0] === 'string' && ActiveComponent.prototype[method] instanceof Function) {
+        return this.findActiveComponentBySource(params[0], (findAcError, ac) => {
+          if (findAcError) {
+            release()
             return cb(err)
```

Comment:
is `err` meant to be `findAcError`?

```suggestion
             return cb(findAcError)
```

Reply:
Yes.